### DISCUSSION TITLE
Option to hide/block the entire new Featured Clips thing.

### DISCUSSION THREAD

**Discussion by Bogdanov89** [night/betterttv#6395] [Ideas]
Please add an option to disable/hide the entire new Featured Clips thing that was added a month or two ago.

Here is a screenshot of the thing:

![image](https://github.com/night/betterttv/assets/55892027/56236514-3539-44c0-ab3a-306d344ddcd9)

**Comment by night** [comment]
Closing this one out — Featured Clips is no longer on the videos tab. Please open a new discussion if it returns or appears elsewhere.
